### ISSUE TITLE
[Feature Request] Add option to display values in 'bits' (hear me out)

### ISSUE THREAD

**Issue by stripey679** [sparrowwallet/sparrow#777]
Summary: Sparrow Wallet should add support for displaying values in bits, in addition to the current options of BTC, sats, and Auto.

Reasoning:
I'll admit, as someone who has grown accustomed to thinking of quantities of bitcoin in sats, when I saw this headline I was skeptical of a proposal of _yet another way to talk about quantities of bitcoin_ (milliBTC or mBTC come to mind as, in my opinion, awkward attempts at named fractional BTC units that aren't very approachable, and I thought bits would be the same). But I read the article, and I think I was convinced the idea has merit. Especially for helping newcomers onboard.

See article for further reasoning and examples of usefulness: https://bitcoinmagazine.com/culture/how-adopting-bits-helps-bitcoin-newcomers

A few highlights that I found persuasive:

> **Assessing Net Worth In Bitcoin**
> 
> Ask anyone with less than a whole bitcoin how much they have, and they will usually reply with the USD (or local currency) equivalent (e.g., I have about $5,000 worth of bitcoin). This figure, of course, will fluctuate with the current price of bitcoin. The chances that they will say, “I have 0.13664418 bitcoin,” is zero. If they are really down the bitcoin rabbit hole, they might say they have “13,664,418 sats.” So, how many more sats do they need to get a whole bitcoin? Just take 13,664,418 from 100,000,000!
> 
> If the bit were to become the accepted norm, they would say “136,644 bits.” There are a million bits in a bitcoin, so the calculation 1,000,000 - 136,644 is so much easier! It’s 863,356, by the way.

 
> **Familiarity With Fiat Currencies**
> 
> To improve the ease of usage of bitcoin, we need to follow some of the principles of traditional fiat currencies such as the dollar, the pound and the euro.
> This can be done without any changes to the core bitcoin principles or architecture and simply relies on a shift in the default denomination of bitcoin. Doing this also positions bitcoin as a more understandable currency to the average person.
> Most traditional fiat currencies are structured in the same way.
> 
> **The Dollar (USD)**
> Of course, $1 dollar equals 100 cents.
> 
> There are no specific names (other than slang) for multiples of the dollar (i.e., 1 dollar, 10 dollars, 100 dollars, 1000 dollars, 100,000 dollars, 1,000,000 dollars).
> 
> The dollar can be written as $1 or as a dollar and cents value, such as $1.57 is 1 dollar and 57 cents.
>
>...
>

> **The Bit (BIT)**
> 
> Similar to the dollar and pound, 1 bit equals 100 sats.
> 
> There does not need to be specific names (other than slang) for multiples of the bit (i.e., 1 bit, 10 bits, 100 bits, 1000 bits, 100,000 bits, 1,000,000 bits).
>
> ...
>

_1,000,000 bits equals 1 BTC_

> [W]e may not need to denominate in whole bitcoin in the future, just 1 million, 2 million, 3 million bits, etc. Just like we do with the dollar, pound and euro. When have you ever heard a USD millionaire referring to their wealth as 100,000,000 cents?

> **Future-Proofing**
> 
> As the price of bitcoin increases, the problem of the perceived affordability of the current denomination of whole bitcoin increases. The bit is well placed to cope with potential bitcoin price increases.

In conclusion, I found the comparison of sats to cents and the proposed 'bits' to dollars or euros to be quite compelling. Having considered that, the idea of having a whole BTC be equal to 1 million bitcoin-version-of-a-dollar units seems much easier to instantly comprehend than a whole BTC being equal to 100 million bitcoin-version-of-a-cent units.

**Comment by RequestPrivacy** [comment]
> The chances that they will say, “I have 0.13664418 bitcoin,” is zero. If they are really down the bitcoin rabbit hole, they might say they have “13,664,418 sats.” So, how many more sats do they need to get a whole bitcoin? Just take 13,664,418 from 100,000,000!
>
> If the bit were to become the accepted norm, they would say “136,644 bits.” There are a million bits in a bitcoin, so the calculation 1,000,000 - 136,644 is so much easier! It’s 863,356, by the way.

Not sure what the author tries to tell us here. He just omitted the last 3 digits for his "bit" calculation vs. the one with sats. So nothing "easier" just "not accurate".

That being said, I get the reasoning. Will I use bits though? No. Should others be able to choose? Why not...

**Comment by lionkmp** [comment]
And working on that, please add "mBTC" too if possible. Thank you!

**Comment by spastad** [comment]
I second this.
Subjective reasoning: I just used to bits.
Objective reasoning: to match Bitcoin Core (and most other wallets).

**Comment by btcpavao** [comment]
I'd also love to be able to display units in bits and sats in decimal places. :)
